bot/seasons/evergreen/movie.py

```diff
@@ -0,0 +1,211 @@
+        self.bot = bot
+        self.http_session: ClientSession = bot.http_session
+
+    @command(name='movies', aliases=['movie'])
+    async def movies(self, ctx: Context, genre: str = "", amount: int = 5) -> None:
+        """
+        Get random movies by specifing genre.
+
+        Also support amount parameter,
+        that define how much movies will be shown. Default 5
+
+        Available genres:
+        - Action
+        - Adventure
+        - Animation
+        - Comedy
+        - Crime
+        - Documentary
+        - Drama
+        - Family
+        - Fantasy
+        - History
+        - Horror
+        - Music
+        - Mystery
+        - Romance
+        - Science
+        - Thriller
+        - Western
+        """
+        # Check is there more than 20 movies specified, due TMDB return 20 movies
+        # per page, so this is max. Also you can't get less movies than 1, just logic
+        if amount > 20:
+            await ctx.send("You can't get more than 20 movies at once. (TMDB limits)")
+            return
+        elif amount < 1:
+            await ctx.send("You can't get less than 1 movies. Just logic.")
+            return
+
+        # Capitalize genre for getting data from Enum, get random page
+        genre = genre.capitalize()
+        page = random.randint(1, 200)
+
+        # Get movies list from TMDB, check is results key in result. When not, raise error. When genre not exist,
+        # show help.
+        try:
+            movies = await self.get_movies_list(self.http_session, MovieGenres[genre].value, page)
+        except KeyError:
+            await ctx.send_help('movies')
+            return
+        if 'results' not in movies.keys():
+            err_text = f'There was problem while fetching movies list. Problematic response:\n```{movies}```'
+            err = Embed(title=':no_entry: Error :no_entry:', description=err_text)
+
+            await ctx.send(embed=err)
+            logger.warning(err_text)
+
+            return
+
+        # Get all pages and embed
+        pages = await self.get_pages(self.http_session, movies, amount)
+        embed = await self.get_embed(genre)
+
+        # Send movies, paginate
+        await ImagePaginator.paginate(pages, ctx, embed)
+
+    async def get_movies_list(self, client: ClientSession, genre_id: str, page: int) -> Dict:
+        """Return JSON of TMDB discover request."""
+        # Define params of request
+        params = {
+            "api_key": TMDB_API_KEY,
+            "language": "en-US",
+            "sort_by": "popularity.desc",
+            "include_adult": "false",
+            "include_video": "false",
+            "page": page,
+            "with_genres": genre_id
+        }
+
+        # Create URL
+        url = BASE_URL + "discover/movie?" + urlencode(params)
+
+        # Make discover request to TMDB
+        async with client.get(url) as resp:
+            data = await resp.json()
+
+        # Return response result
+        return data
```

Comment:
`data` can be returned directly like here for consistency https://github.com/python-discord/seasonalbot/blob/cce99c584364ea749a542d5c8fcbe2af8ea3c075/bot/seasons/evergreen/movie.py#L165-L166